zml/zml.zig

```diff
@@ -29,7 +29,8 @@ pub const pjrt = @import("pjrtx.zig");
 pub const testing = @import("testing.zig");
 pub const torch = @import("torch.zig");
 
-pub const tokenizer = @import("tokenizer.zig");
+// pub const tokenizer = @import("tokenizer.zig");
+pub const tokenizer = @import("zml/tokenizer");
```

Comment:
could we move "tokenizer" to it's own library so that not all of zml requires a Rust toolchain ?